app/controllers/wrangling_guidelines_controller.rb

```diff
@@ -13,57 +15,64 @@ def show
 
   # GET /wrangling_guidelines/new
   def new
+    authorize :wrangling if logged_in_as_admin?
```

Comment:
All of these actions are admin only (see the `before_action`), so the `logged_in_as_admin?` check is redundant. Please remove it here and from the other authorize calls in this controller.
 ```suggestion
     authorize :wrangling
```